configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/ochuman/hrnet_ochuman.yml

```diff
@@ -0,0 +1,70 @@
+Collections:
```

Comment:
The metafile format has been updated in #941. Please re-run the pre-commit hook after rebasing onto master.